src/wyzeapy/services/garage_service.py

```diff
@@ -0,0 +1,52 @@
+from .base_service import BaseService
+from ..types import Device, DeviceTypes
+
+class GarageDoor(Device):
+    open = False
+    closing = False
+    opening = False
+
+class GarageService(BaseService):
+    async def update(self, garage: GarageDoor):
+        async with BaseService._update_lock:
+            garage.device_params = await self.get_updated_params(garage.mac) 
+
+        device_info = await self._get_garage_info(garage)
```

Comment:
Where is this function?

Reply:
Not pushed! :D Checked my git output and figured out my GPG key expired and it never actually committed. The one time I use IDE git integrations...